libraries/AP_Notify/AP_Notify.h

```diff
@@ -28,6 +28,7 @@
 #define BUZZER_ON       1
 #define BUZZER_OFF      0
 
+#define NOTIFY_TEXT_NUM_ROWS 6
```

Comment:
Should be in the backend, we might support a different number of rows in the future.

Reply:
Bump on this.

Reply:
I believe its sitting where it needs to be which is next to NOTIFY_TEXT_BUFFER_SIZE. NOTIFY_TEXT_BUFFER_SIZE is equivalent to say Column size. That variable is being used in various places outside of AP_Notify, so cannot be moved right now.

At some stage that may need cleanup or maybe it is fine the way it is. The fate of both NOTIFY_TEXT_BUFFER_SIZE and NOTIFY_TEXT_NUM_ROWS needs to be decided together, so they need to stay together.

Reply:
odd, its used in MSP_Telem and OSD, but the number of rows could still be in Display.h